### PR TITLE
Remove Beacon Files from Project and Remove Resigning of FirstResponder for GREYKeyboard to turn off AutoCorrection.

### PR DESCRIPTION
**Pull request by tirodkar**
For https://github.com/google/EarlGrey/issues/56.

### CODE REVIEW

File: EarlGrey/Action/GREYActions.m [hunk 1]
```diff
@@ -167,56 +167,60 @@ @implementation GREYActions
...
-
     // If a position is given, move the text cursor to that position.
     if (position) {
       UITextRange *newRange = [element textRangeFromPosition:position toPosition:position];
       [element setSelectedTextRange:newRange];
     }
 
-    // After autocorrect is disabled, we can perform the actual typing.
-    BOOL retVal = [GREYKeyboard typeString:text error:errorOrNil];
+    BOOL retVal = NO;
```
Comment (khandpur) [inline EarlGrey/Action/GREYActions.m]: nit: you don't need to assign to NO. it's the default.

File: EarlGrey/Action/GREYActions.m [hunk 2]
```diff
@@ -167,56 +167,60 @@ @implementation GREYActions
...
+    } else {
+      // Perform typing. If this is pre-iOS8.2, then we simply turn the autocorrection
+      // off the current textfield being typed in.
+      retVal = [GREYActions grey_disableAutoCorrectForDelegateAndTypeText:text
+                                                                withError:errorOrNil];
     }
 
-    return retVal;
-  }];
+      return retVal;
```
Comment (khandpur) [inline EarlGrey/Action/GREYActions.m]: indentation seems off here.

File: EarlGrey/Common/GREYExposed.h [hunk 3]
```diff
@@ -307,3 +319,28 @@ IOHIDEventRef IOHIDEventCreateDigitizerFingerEvent(CFAllocatorRef allocator,
...
+- (void)setAutocorrectionPreferenceValue:(NSNumber *)enabled forSpecifier:(id)specifier;
+
+/**
+ *  Removes the predictive text from being accepted on a UIKeyboard to stop
+ *  displaying the autocorrect options when typing.
+ *
+ *  @param enabled   A boolean that indicates if predictive typing is enabled or not.
+ *  @param specifier Is unused can be @c nil.
+ */
+- (void)setPredictionPreferenceValue:(id)arg1 forSpecifier:(id)arg2;
```
Comment (khandpur) [inline EarlGrey/Common/GREYExposed.h]: @param comment and the parameter name don't match

**Comment by khandpur** [comment]
There are some keyboard tests that are failing on Travis which are related to this PR:
https://travis-ci.org/google/EarlGrey/jobs/119667558
https://travis-ci.org/google/EarlGrey/jobs/119667559

**Comment by khandpur** [comment]
fyi, this breaks some keyboard tests and is on hold.

**Comment by tirodkar** [comment]
The Keyboard tests seem to be fixed in the latest CI run.

**Pull request merged**